hardware_interface/include/hardware_interface/joint_command_handle.hpp

```diff
@@ -55,6 +55,9 @@ class JointCommandHandle
   void
   set_cmd(double cmd);
 
+  HARDWARE_INTERFACE_PUBLIC
+  bool valid_pointers() const;
```

Comment:
that function name should be more precise. `valid_pointers` doesn't really say anything about what that function does, meaning which pointers are being checked.

Reply:
also, the function states `pointers` as in multiple pointers, the implementation does however only check a single `cmd_`.

Reply:
I have the same function implemented in all the other handles and I call this from the template function so they should all be called the same. TBH a proper solution would see inheriting such an interface to really ensure this function existing the same way for all